azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -68,21 +59,7 @@ def configure_azure_monitor(**kwargs) -> None:
     end user to configure OpenTelemetry and Azure monitor components. The
```

Comment:
Do we still need `InstrumentationConfig` type above?

Reply:
Good point. Will remove.

Reply:
Done